docs/website/blog/3.0.0-GA-blog.md

```diff
@@ -0,0 +1,333 @@
+
+Read this section on [What happened to Ingress/Route?](/docs/user-guides/v3-migration-guide#what-happened-to-ingressroute) to learn more.
+
+### odo add binding
+
+`odo add binding` allows the application to connect to an Operator Backed service.
+
+```shell
+$ odo add binding
+? Do you want to list services from: current namespace
+? Select service instance you want to bind to: cluster-sample (Cluster.postgresql.k8s.enterprisedb.io)
+? Enter the Binding's name: my-node-app-cluster-sample
+? How do you want to bind the service? Bind As Files
+? Select naming strategy for binding names: DEFAULT
+ ✓  Successfully added the binding to the devfile.
+Run `odo dev` to create it on the cluster.
+You can automate this command by executing:
+  odo add binding --service cluster-sample.Cluster.postgresql.k8s.enterprisedb.io --name my-node-app-cluster-sample
+```
+
+_How is it different from `odo link` command of v2?_
+* [Interactive mode](#interactive-mode)
+* [Ability to connect to a service in any namespace](#ability-to-connect-to-a-service-in-any-namespace)
+* [Run from anywhere](#run-from-anywhere)
+* [Work without a running application](#work-without-a-running-application)
+* [Only supports Bindable Operators](#only-supports-bindable-operators)
+* [Requires the Service Binding Operator](#requires-the-service-binding-operator)
+
+#### Interactive mode
+`odo add binding` provides a very nice interactive mode that lists bindable service across all the namespaces, and provides a way to personalize related configuration, making the experience nicer.
+
+#### Ability to connect to a service in any namespace
+`odo add binding` allows the application to connect to any bindable service in any namespace, it does not necessarily have to be in the same namespace as the application.
+
+#### Run from anywhere
+`odo add binding` can be run from anywhere and does not require access to a Devfile in order to work. With this method, you can either directly create the Service Binding on the cluster, write the YAML to a file, or simply display it.
+
+#### Work without a running application
+As opposed to v2, `odo add binding` does not require for the application be running on the cluster, making it even faster.
+
+#### Only supports Bindable Operators
+
+`odo link` could link an application to another application and any Operator Backed service, but as of now, v3 only supports connecting to services that are backed by Operators considered [bindable](https://github.com/redhat-developer/service-binding-operator#known-bindable-operators) by the Service Binding Operator.
+
+Based on the user feedback, we might work on adding these features back, but as of now there are no plans of implementing them.
+
+Regardless, there are workarounds to connect to another application
+or non-bindable service, read [this blog](https://odo.dev/blog/binding-external-service-with-odo-v3) to know how.
```

Comment:
For all links to [odo.dev](https://odo.dev) in this article, I think it would make more sense to use relative URL paths (like `../binding-external-service-with-odo-v3` here)? This makes the links more portable, IMO.

Reply:
I usually use relative URL paths, I must've missed this one. Thanks for catching!

Reply:
You're welcome. FYI, there are a few other similar absolute links in the article.